lib/galaxy/config/sample/themes_conf.yml.sample

```diff
@@ -46,3 +60,17 @@ pride:
       logo:
         img: "/static/favicon.svg"
         img-secondary: null
+
+smoky:
+  masthead:
+    color: "#0C0F0A"
+    text:
+      color: white
+      hover: "#FBFF12"
+      active: white
+    link:
+      color: transparent
+      hover: transparent
+      active: "#FF206E"
+    logo:
+      img: "/static/favicon.svg"
```

Comment:
This example theme demonstrates the same thing as `lightblue`. If we add a fourth example theme, I think it should showcase something not done by the other themes. E.g: setting an image background, or chaining the logo.

Reply:
I am open for suggestions and have no preference regarding any of the themes. I think it would be best to do this separately i.e. unrelated to this PR, and let the community pick/vote four themes.